deploy/scheduler/deployment.yaml

```diff
@@ -98,3 +99,7 @@ data:
           - name: KnodeVolumeBinding
             args:
               bindTimeoutSeconds: 5
+        #  - name: KNodeDistribution
+        #    args:
+        #    # kubeConfigPath: "REPLACE_ME_WITH_KUBE_CONFIG_PATH"
+        #      kubeConfigPath: "/etc/kubernetes/scheduler.conf"
```

Comment:
The code here is in comment status, is it normal?

Reply:
yes~,You can refer to [scheduler-plugins](https://github.com/kubernetes-sigs/scheduler-plugins/blob/release-1.21/manifests/noderesourcetopology/scheduler-config.yaml#L6)